packages/backend/src/modules/tvl2/repositories/AmountRepository.ts

```diff
@@ -32,8 +32,19 @@ export class AmountRepository extends BaseRepository {
     const rows = await knex('amounts')
       .whereIn('configuration_id', configIds)
       .where('timestamp', timestamp.toDate())
+      .andWhereRaw(`extract(hour from "timestamp") % 24 = 0`)
```

Comment:
this change will modify the /api/tvl behaviour - is it intended?

Reply:
I think this will modify the `/api/tvl2`. We do not use it anywhere, so it does not matter. This will fix a bug there.